### PR TITLE
april dev update

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: content/posts/irods-development-update-april-2024.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+Author: Kory Draughn
+Slug: irods-development-update-april-2024
+Status: published
+
+
+Hello reader,
+
+We're more than half-way through April now, which means the [iRODS User Group Meeting](https://irods.org/ugm2024) is right around the corner. If you're curious about iRODS, consider registering for the conference. We hope to see you there.
+
+iRODS 4.3.2 is inching closer to a release. The next release of the server brings several deprecations: GeneralUpdate API, RBUDP, msiSendMail, msiSetResource, and BACKUP_RESC_NAME_KW. Now only that, but the GenQuery2 parser has been absorbed into the server and will ship with the 4.3.2 release. You may be wondering if the GenQuery2 parser is used within the server and the answer is ... not at this time. There are plans to start using it within the server and plugins, but only after confidence is built and that takes time.
```
Comment (trel) [inline content/posts/irods-development-update-april-2024.md:12]: "Now only that" -> "Not only that"

**Review by trel — COMMENTED** [review]
one thing...

File: content/posts/irods-development-update-april-2024.md (hunk 1 above)
Comment (korydraughn) [reply content/posts/irods-development-update-april-2024.md]: Fixed.

**Comment by korydraughn** [comment]
Squashed.

**Pull request merged**